04week/welcome-to-our-site/index.html

```diff
@@ -8,67 +8,41 @@
 </head>
 
 <body>
-
-  <!-- Instead of a NavBar, I wanted this to be a banner instead -->
-  <div class="TopBar">
-    <img src="./img/epic-logo.jpg" class="Logo" />
-    <img src="./img/fortnite-banner.jpg" class="Banner" />
+  <div class="Welcome">
+    <h1>Welcome to our site</h1>
   </div>
-  <!-- Next I wanted a semi-transparent information panel on the left and
-  an "Epic Games signup" on the right-->
-  <div class="BodyParagraphs">
```

Comment:
try not to start classnames or ids with capital letters.. instead use camelCase

i.e. `class="bodyParagraphs"`